### ISSUE TITLE
Date Conversion Issue

### ISSUE THREAD

**Issue by subrat7** [clarketm/TableExport#111]
Hi

I am using Tableexport version(5.0.0),i have date in the table and while export to excel i am facing below issue

Html table data
![image](https://user-images.githubusercontent.com/12031027/36407923-1fcc96aa-1628-11e8-8360-c6285a06c9ca.png)

and excel data is
![image](https://user-images.githubusercontent.com/12031027/36372961-ae8b0b16-158c-11e8-85b9-7feb9307519f.png)
please help me how to resolve this issue,Thanks in Advance

**Comment by joaociocca** [comment]
Now I'm thinking this could be related to what I mentioned in https://github.com/clarketm/TableExport/issues/124#issuecomment-376320828

**Comment by StuffieStephie** [comment]
Before you call an instance of `TableExport` to the following:
`
TableExport.prototype.typeConfig.date.assert = function(value){return false;};
`
I had a similar issue with P.O. Box numbers being converted to dates.

**Comment by brunodotcom** [comment]
I've had a similar problem with text that was being converted to date without reason.
But @StuffieStephie answer fixed my code, thanks a lot!

**Comment by nowherenearithaca** [comment]
yeah - I just disabled it trying to convert dates as well as described by @StuffieStephie

**Comment by barman-a** [comment]
Hi,

I am having the same issue in date conversion. It seems that solution from StuffieStephie is working for all, however, I am a bit confused where to add this in the code. My code looks like below.

$(function(){
$('#<?php echo $tid1; ?>').tableExport({
	        formats: ["xlsx", "csv"],
		bootstrap: true,
	        sheetname: "sheet1",
		position: top,
		ignoreRows: null
   });
   });
